tutorials/nginx-ingress-gke/index.md

```diff
@@ -0,0 +1,353 @@
+   which allows you to rewrite the URI of a request before sending it to the
+   application.
+-  [Session Persistence](https://github.com/nginxinc/kubernetes-ingress/blob/master/examples/session-persistence)
+   (NGINX Plus only), which guarantees that all the requests from the same
+   client are always passed to the same backend container.
+-  [Support for JWTs](https://github.com/nginxinc/kubernetes-ingress/blob/master/examples/jwt)
+   (NGINX Plus only), which allows NGINX Plus to authenticate requests by
+   validating JSON Web Tokens (JWTs).
+
+The following diagram shows the architecture described above:
+
+<img src="https://github.com/ameer00/community/blob/master/tutorials/nginx-ingress-gke/Nginx%20Ingress%20on%20GCP%20-%20Fig%2002.png" width="65%">
+
+This tutorial illustrates how to set up a
+[deployment in Kubernetes](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+with an Ingress Resource using nginx as the ingress controller to route/load
+balance traffic from external clients to the Deployment.  With this tutorial,
+admins will be able to:
+
+-  Set up an application deployment in Kubernetes
+-  Set up an Ingress Resource object for the Deployment
+-  Set up an Ingress Controller using nginx as the controller (instead of the
+   GKE default Cloud Load Balancer offered by GCP)
+
+# Objectives
+
+-  Create a simple Kubernetes web _application_ deployment 
+-  Create a _default backend_ deployment to serve 404 for all non-matching
+   hosts and paths
+-  Create an _Ingress Resource_ for the _application_
+-  Configure an _nginx ingress_ deployment as an Ingress Controller for the
+   _application_
+-  Test nginx ingress functionality by accessing the Google Cloud L4 LB
+   frontend IP and ensure it can access the web application
+
+# Costs
+
+This tutorial uses billable components of Cloud Platform, including:
+
+-  Google Kubernetes Engine
+-  Google Cloud Load Balancing
+
+Use the [Pricing Calculator](https://cloud.google.com/products/calculator) to
+generate a cost estimate based on your projected usage.
+
+# Before You Begin
+
+1. You'll need a Google Cloud Platform (GCP) project. Select or create a GCP project.  
+   [GO TO THE PROJECTS PAGE](https://console.cloud.google.com/project)
+   
+1. Enable billing for your project.  
+	[ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
+
+1. Enable the Google Kubernetes Engine (GKE) API.  
+	[ENABLE APIs](https://console.cloud.google.com/flows/enableapi?apiid=container,cloudresourcemanager.googleapis.com)
+
+# Set up your environment
+
+In this section you configure the infrastructure and identities required to
+complete the tutorial.
+
+## Start a Cloud Shell instance and create a Kubernetes Engine cluster
+
+1. You'll run all the terminal commands in this tutorial from Cloud Shell.  
+	[OPEN CLOUD SHELL](https://console.cloud.google.com/?cloudshell=true)
```

Comment:
Add this sentence under the heading:

"You can use Cloud Shell to complete this tutorial. To use Cloud Shell, perform the following steps:"

Change to a link with the call to action:

[Open a new Cloud Shell session]